online-shop/.editorconfig

```diff
@@ -0,0 +1,14 @@
+
```

Comment:
Please be more careful when doing commits: never commit the node_modules folder and don't commit directly on master.